mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -289,7 +289,10 @@ def _on_stack_reconstruct_slice_done(self, task: TaskWorkerThread):
             slice_idx = self._get_slice_index(None)
             if images is not None:
                 assert self.model.images is not None
-                images.name = "Recon"
+                filter_name = self.view.recon_params().filter_name
+                algorithm_name = self.view.recon_params().algorithm
+                images.name = f"Recon_Slice{algorithm_name}"
+                images.name = f"{images.name}_{filter_name}" if filter_name else images.name
```

Comment:
The filter should only be added for gridrec and FBP, its not used by the iterative methods.